source/code/scxsystemlib/common/scxostypeinfo.cpp

```diff
@@ -52,7 +52,7 @@ using namespace SCXCoreLib;
 
 namespace {
 
-#if defined(linux)
+#if defined(linux) && defined(PF_DISTRO_SUSE)
```

Comment:
Got it!

Yeah, this is probably okay. But if you do this, you should also remove it from the header file. And you absolutely need to test it in PBUILD (or test it on Ubuntu 15 and ask me to test other platforms).

The problem here is that we only build on what we build on. For Linux, we don't generally build on Ubuntu 15 (or any Ubuntu, for that matter). We build FOR Ubuntu, but not ON Ubuntu. We will eventually find this as we build on some newer platform with some newer compiler, but that can take a while.

In any case, if you'd like, fix the header file (so if there are references, you'll find out earlier), make a new shelveset, test on all platforms that you can easily test on, and then ask me to PBUILD it for the remainder (mentioning the name of your shelveset). Thanks!